templates/vue/package.json

```diff
@@ -31,6 +31,7 @@
     "bootstrap-vue": "^2.13.1",
     "d3": "^5.16.0",
     "dragselect": "^1.13.1",
+    "jquery": "^3.5.1",
```

Comment:
@shirleywbi I don't think we actually need to include this in our build since Wordpress already loads jQuery and this functionality is dependent on Wordpress. Was there a specific reason you included it?

Reply:
@wynnset I think you're right. Just tested it without the package and it works fine. Removing it.